### ISSUE TITLE
Typescript for winston Logger

### ISSUE THREAD

**Issue by huycuong89** [moleculerjs/moleculer#434]
## Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [x] I checked the documentation and found no answer

## Expected Behavior

When add "winston" as logger in broker's options, it should be allowed.

## Current Behavior
"winston" not typed as "Logger", Typescript throw error.

## Failure Information

The expected type comes from property 'logger' which is declared here on type 'BrokerOptions'

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Create "winston" logger
2. Add "winston" to broker's options

### Reproduce code snippet
```js
const loggerWinter: WTLogger = winston.createLogger({
  level: 'info',
  format: winston.format.combine(winston.format.timestamp(), winston.format.json()),
  transports: [
    new winston.transports.File({ filename: 'error.log', level: 'error' }),
    new winston.transports.File({ filename: 'combined.log' }),
  ],
});

const brokerConfig: BrokerOptions = {
  namespace: '',
  nodeID: (NODEID ? NODEID + '-' : '') + os.hostname().toLowerCase(),

  logger: loggerWinter, // WARNING FROM TYPESCRIPT
  logLevel: 'info',
  logFormatter: 'default',
  // other options
}
```

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Moleculer version: 0.13
* NodeJS version: 10.13.0
* Operating System: Ubuntu 18.04 x64

**Comment by darky** [comment]
tablet for this here: https://github.com/moleculerjs/moleculer/pull/501/

**Comment by darky** [comment]
@icebob, it can be closed.
Will be fixed in version `> 0.13.8`

**Comment by darky** [comment]
Example: https://moleculer.services/docs/0.13/logging.html#Winston
